presentation/testing/src/presentation-testing/IModelUtilities.ts

```diff
@@ -6,84 +6,89 @@
  * @module IModel
  */
 
-import { IModelDb, IModelJsFs, InformationPartitionElement, SnapshotDb } from "@itwin/core-backend";
+import { IModelDb, IModelJsFs, SnapshotDb } from "@itwin/core-backend";
 import { Id64String } from "@itwin/core-bentley";
-import { Code, CodeScopeProps, CodeSpec, ElementAspectProps, ElementProps, IModel, LocalFileName, ModelProps } from "@itwin/core-common";
+import { BisCodeSpec, Code, CodeScopeProps, CodeSpec, ElementAspectProps, ElementProps, LocalFileName, ModelProps } from "@itwin/core-common";
 import { IModelConnection, SnapshotConnection } from "@itwin/core-frontend";
 import path from "path";
-import { TEST_OUTPUT_DIR } from "./Helpers";
+import { getTestOutputDir } from "./Helpers";
 
+/**
+ * Interface for IModel builder pattern. Used for building IModels to test rulesets.
+ *
+ * @public
```

Comment:
Let's make new APIs `@beta`

Reply:
Marked interface as `@beta`. Should I do the same with the implementation?

Reply:
No, we're not expecting the implementation to be used directly by consumers.